lib/bolt_ext/schemas/ssh-run_task.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
I saw these defined in the puppet repo (https://github.com/puppetlabs/puppet/blob/6597624ccfc79ce8f8f0632defd5587d1c026b4b/api/schemas/catalog.json#L2), I think that `json-schema` gem assumes draft-04 by default. Will this cause a problem if there is no internet connection to json-schema.org?

Reply:
It appears the `$schema` here is just parsed to determine which of the built-in validators to use, so no network required.